ape_frame/accounts.py

```diff
@@ -36,8 +34,11 @@ def wrap_sign(fn: Callable) -> Optional[bytes]:
     try:
         return fn()
 
-    except ValueError as err:
-        if not err.args[0]["message"] == "User declined transaction":
+    except Web3RPCError as err:
```

Comment:
Think this change was made in web3py v7, such that it does not inherit from `ValueError` anymore